scripts/automation/regression/trex_unit_test.py

```diff
@@ -513,8 +516,8 @@ def begin (self):
         if not self.no_daemon:
             if self.kill_running:
                 client.kill_all_trexes()
-            elif client.get_trex_cmds():
-                print('TRex is already running. Use --kill-running flag to kill it.')
+            elif client.get_trex_cmds() and not self.allow_multi_instance:
```

Comment:
Please rewrite as:
elif (not self.allow_multi_instance) and client.get_trex_cmds()
(for optimization)